src/System.Net.NameResolution/tests/PalTests/System.Net.NameResolution.Pal.Tests.csproj

```diff
@@ -56,14 +56,20 @@
     <Compile Include="$(CommonTestPath)\System\Net\Configuration.Http.cs">
       <Link>Common\System\Net\Configuration.Http.cs</Link>
     </Compile>
-	<Compile Include="$(CommonPath)\System\Net\ByteOrder.cs">
+    <Compile Include="$(CommonPath)\System\Net\ByteOrder.cs">
       <Link>Common\System\Net\ByteOrder.cs</Link>
     </Compile>
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
     <Compile Include="..\..\src\System\Net\NameResolutionPal.Windows.cs">
       <Link>ProductionCode\System\Net\NameResolutionPal.Windows.cs</Link>
     </Compile>
+    <Compile Include="..\..\src\System\Net\NameResolutionPal.Windows.Uap.cs" Condition="'$(TargetGroup)' == 'uap'">
```

Comment:
Nit: I think this should be just `NameResolutionPal.Uap.cs` to follow the convention

Reply:
Fixed